docs/tutorial/build_the_content_list.md

```diff
@@ -0,0 +1,441 @@
+    Y.namespace('eZConf');
+
+    Y.eZConf.ListView = Y.Base.create('ezconfListView', Y.eZ.TemplateBasedView, [], {
+        initializer: function () {
+            console.log("Hey, I'm the list view");
+        },
+
+        render: function () {
+            this.get('container').setHTML(
+                this.template({
+                    locations: this._getJsonifiedLocations(),
+                })
+            );
+            return this;
+        },
+
+        _getJsonifiedLocations: function () {
+            // to get usable objects in the template
+            return Y.Array.map(this.get('locations'), function (loc) {
+                return loc.toJSON();
+            });
+        },
+    }, {
+        ATTRS: {
+            locations: {
+                value: [],
+            }
+        }
+    });
+});
+```
+
+Then the template has to be changed to something like:
+
+``` html
+<!-- **ezconflistview.hbt** -->
+
+<h1 class="ezconf-list-title">List view</h1>
+
+<ul class="ezconf-list">
+{{#each locations}}
+    <li><a href="{{path 'viewLocation' id=id languageCode=contentInfo.mainLanguageCode}}">{{ contentInfo.name }}</a></li>
+{{/each}}
+</ul>
+```
+
+PlatformUI provides a `path` template helper that allows you to generate a route URL in PlatformUI. It expects a route name and the route parameters.
+
+##### Results
+
+The resulting code can be been in [the `6_1_list_client` tag on GitHub](https://github.com/ezsystems/ExtendingPlatformUIConferenceBundle/releases/tag/6_1_list_client), this step result can also be viewed as [a diff between tags `5_navigation` and `6_1_list_client`](https://github.com/ezsystems/ExtendingPlatformUIConferenceBundle/compare/5_navigation...6_1_list_client).
+
+The rest of this tutorial is focused on the server side rendering strategy. Completing the browser side rendering strategy to get the expected features is left as an exercise.
+
+## Server side rendering
+
+In this case a part of the rendering is delegated to the server. When building a PlatformUI page this way, the application will just do one or more AJAX request(s) and inject the result in the UI. The PlatformUI *Admin part* is built this way. To be easily usable in the JavaScript application, the server response has to be structured so that the application can retrieve and set the page title, the potential notifications to issue and the actual page content. This is done by generating an HTML fragment, in the following way:
+
+``` html
+<!-- Example of server side response -->
+
+<div data-name="title">Title to set in the application</div>
+<div data-name="html">
+    <p>Page content</p>
+</div>
+<ul data-name="notification">
+    <li data-state="done">I'm a "done" notification to display in the application</li>
+</ul>
+```
+
+### View service to fetch an HTML fragment
+
+#### Minimal view service
+
+In the case, the minimal view service is exactly the same as the one produced in [the previous Minimal view service paragraph](#minimal-view-service).
```

Comment:
Yes, I checked the link will work. If there are two identical headings in one file, the automatic anchors to next headings are appended a number.